src/notation/internal/notationselectionrange.cpp

```diff
@@ -140,12 +140,12 @@ Ms::Segment* NotationSelectionRange::rangeStartSegment() const
 {
     Ms::Segment* startSegment = score()->selection().startSegment();
 
-    startSegment->measure()->firstEnabled();
-
     if (!startSegment) {
         return nullptr;
     }
 
+    startSegment->measure()->firstEnabled();
```

Comment:
outch, another good catch!

Reply:
Note to self: the only change of this PR that won't apply to 3.x, as far as I can tell)